resources/lib/kodi/context_menu.py

```diff
@@ -27,15 +27,15 @@ def ctx_url_builder(videoid):
     'update': {
         'label': common.get_local_string(30061),
         'url': ctx_item_url(['update'], g.MODE_LIBRARY)},
-    'exportnewepisodes': {
+    'export_new_episodes': {
         'label': common.get_local_string(30195),
-        'url': ctx_item_url(['exportnewepisodes'], g.MODE_LIBRARY)},
-    'excludefromautoupdate': {
+        'url': ctx_item_url(['export_new_episodes'], g.MODE_LIBRARY)},
+    'exclude_from_auto_update': {
         'label': common.get_local_string(30196),
-        'url': ctx_item_url(['excludefromautoupdate'], g.MODE_LIBRARY)},
-    'includeinautoupdate': {
+        'url': ctx_item_url(['exclude_from_autou_pdate'], g.MODE_LIBRARY)},
```

Comment:
Small typo leading to an error ^^ 
Should be exclude_from_auto_update

Reply:
🚀